packages/core/src/shared/errors.ts

```diff
@@ -333,7 +334,6 @@ export function getTelemetryResult(error: unknown | undefined): Result {
  */
 export function scrubNames(s: string, username?: string) {
     let r = ''
-    const fileExtRe = /\.[^.]{1,4}$/
```

Comment:
probably doesn't matter, but would keeping this allow us to avoid the 'path' import?

Reply:
Looked back in to the pattern as it was the root for the other failing tests. A small fix and now we can avoid path import.